src/main/java/pl/olita/openweathermap/WeatherService.java

```diff
@@ -0,0 +1,37 @@
+package pl.olita.openweathermap;
+
+import org.springframework.stereotype.Service;
+import org.springframework.web.client.RestTemplate;
+
+@Service
+public class WeatherService {
+
+    private static final String URL_WEATHER = "http://api.openweathermap.org/data/2.5/weather?q=";
+    private static final String URL_API_KEY = "&units=metric&appid=1ef46e80302a68f49c545c03d505606c";
```

Comment:
Jak tak patrzę na te stringi to myślę że bardziej pobawił się obiektem URI - będzie wg mnie czytelniej - oczywiśćie nie narzucam sposobu